### DISCUSSION TITLE
Unable to resize using mouse only if no border

### DISCUSSION THREAD

**Discussion by c02y** [i3/i3#5015] [Q&A] [answered]
I'm not sure if this is a bug or not, but when there is no border for floating window or two windows in split layout, resizing the windows using only mouse will not work.

If this is not designed on purpose, then it is a bug, then I'll open a bug issue and provide the needed info.

**Comment by Airblader** [comment, marked as answer]
How are you trying to resize? There's two ways to resize with the mouse, either by dragging the border or by pressing the modifier key and then click dragging. The former doesn't work on borderless windows for obvious reasons.

**Reply by c02y** [reply]
`dragging the border`, so I believe this is designed like this instead of a bug.

**Reply by c02y** [reply]
By default, I can use right-click on the window bar and drag to resize the floating window, and it seems I cannot change the right-click to another mouse button, but I already use this button on window bar to kill the window in my config, I'll use another mouse button to kill the window then.
